Feature/gherkin v6 rule keyword

nbossard opened the pull request:
Hello, This pull request is for adding coloring of new keyword "rule", introduced in Gherkin V6 https://cucumber.io/docs/gherkin/reference/#rule
Best regards, Nicolas BOSSARD.

tpope commented:
I've updated the dictionary with the latest keywords. Build on top of that and I can merge this.

nbossard commented:
Great, thanks.

nbossard commented:
Hello, I have rebased branch on your latest master, removed my changes on dictionnary. Should be mergeable now.
Best regards, Nicolas.
Rem : this MR only updates syntax file, working on indent file, coming soon hopefully.
